### PR TITLE
React Router DOM: `MatchOptions`

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky @sgrishchenko

### CODE REVIEW

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/matchPath.kt [hunk 1]
```diff
@@ -6,5 +6,6 @@ package react.router.dom
 @JsName("matchPath")
 external fun rawMatchPath(
     pathName: String,
-    options: dynamic,
+    props: RouteProps,
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/matchPath.kt:9]: Invalid signature

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/matchPath.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-react-router-dom/src/main/kotlin/react/router/dom/matchPath.kt]: Fixed

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/MatchOptions.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@file:JsModule("react-router-dom")
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/MatchOptions.kt:1]: Redundant

Comment (aerialist7) [reply kotlin-react-router-dom/src/main/kotlin/react/router/dom/MatchOptions.kt]: Fixed

**Review by turansky — APPROVED** [review]

**Comment by Leonya** [comment]
Released in pre.240

**Pull request merged**